python/rascaline/rascaline/utils/clebsch_gordan/correlate_density.py

```diff
@@ -46,11 +63,6 @@ def correlate_density(
     controlled with arguments `angular_cutoff`, `angular_selection` and
     `parity_selection`.
 
-    :param density: A density descriptor of body order 2 (correlation order 1),
-        in :py:class:`TensorMap` format. This may be, for example, a rascaline
-        :py:class:`SphericalExpansion` or :py:class:`LodeSphericalExpansion`.
-        Alternatively, this could be multi-center descriptor, such as a pair
-        density.
     :param correlation_order: The desired correlation order of the output
```

Comment:
```suggestion
    :param max_angular: The maximum angular of the density after the CG iterations. This is important for the precomputation of the CG coefficients. It should be :param angular cutoff: if it is not None, otherwise it is ``max_angular`` of the input ``density`` in the compute function times ``correlation_order``.
    :param correlation_order: The desired correlation order of the output
```